lib/cc/service/http.rb

```diff
@@ -59,6 +58,10 @@ def http_method(method, url = nil, body = nil, headers = nil)
       req.headers.update(headers) if headers
       req.body = body if body
       block.call req if block
+
+      unless CC::Service::SafeWebhook.safe?(url)
+        raise "Invalid Webhook"
```

Comment:
Seems like we should have a class for this exception and not use `StandardError`?